src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

Comment:
7'th point in checklist

Reply:
test report link file:///C:/Users/User/layout_search-bar-airbnb/backstop_data/html_report/index.html